packages/core/src/codewhisperer/client/codewhisperer.ts

```diff
@@ -265,6 +265,8 @@ export class DefaultCodeWhispererClient {
         }
         const response = await (await this.createUserSdkClient()).sendTelemetryEvent(requestWithCommonFields).promise()
         getLogger().debug(`codewhisperer: sendTelemetryEvent requestID: ${response.$response.requestId}`)
+
+        return response
```

Comment:
is this change necessary?

Reply:
i prefer having these so that callee can do whatever they want for the response like logging or something else

Reply:
OK I am fine as long as this does not cause regression